crates/polars-time/src/windows/duration.rs

```diff
@@ -412,32 +412,32 @@ impl Duration {
     /// For example, 2022-11-06 01:30:00 CST truncated by 1 hour becomes 2022-11-06 01:00:00 CST,
     /// whereas 2022-11-06 01:30:00 CDT truncated by 1 hour becomes 2022-11-06 01:00:00 CDT.
     ///
-    /// * `original_dt_naive` - original datetime, without time zone.
+    /// * `original_dt_local` - original datetime, without time zone.
     ///   E.g. if the original datetime was 2022-11-06 01:30:00 CST, then this would
     ///   be 2022-11-06 01:30:00.
     /// * `original_dt_utc` - original datetime converted to UTC. E.g. if the
     ///   original datetime was 2022-11-06 01:30:00 CST, then this would
     ///   be 2022-11-06 07:30:00.
-    /// * `result_dt_naive` - result, without time zone.
+    /// * `result_dt_local` - result, without time zone.
     #[cfg(feature = "timezones")]
     fn localize_result(
         &self,
-        original_dt_naive: NaiveDateTime,
+        original_dt_local: NaiveDateTime,
         original_dt_utc: NaiveDateTime,
```

Comment:
drive-by: improve naming (both `original_dt_naive` and `original_dt_utc` are `NaiveDateTime`, so this should reduce confusion. The docstring above the function definition explains the difference)